tao_compiler/mlir/disc/disc_compiler_main.cc

```diff
@@ -136,6 +136,15 @@ int InitCuda(GpuDeviceInfo& ctx) {
   RETURN_ON_CUDA_ERROR(
       cuDeviceComputeCapability(&ctx.cc_major, &ctx.cc_minor, device),
       "cuDeviceComputeCapability");
+  RETURN_ON_CUDA_ERROR(
+      cuDeviceGetAttribute(&ctx.sm_count,
```

Comment:
This is for CUDA GPU only. So the sm_count and max_threads_per_sm will be default value (-1) for ROCM GPU until we add the supplements in the ROCM part?
cc @lfengad